src/hardware/cpe-ubiquiti.md

```diff
@@ -0,0 +1,117 @@
+# CPE - Ubiquiti
+
+This document covers setup for the following devices. Other devices may be similar.
+
+- [Loco5AC](https://www.ui.com/airmax/nanostation-ac/)
+- [LBE‑5AC‑23](https://www.ui.com/airmax/litebeam-ac/)
+
+## What you will need
+
+The following information you will need to collect before beginning  the configuration.
+
+- SSID of the Supernode antenna you wish to connect to
+- Your assigned node name
+- Your assigned IP address, Netmask, Gateway
+- A computer with WiFi for management radio method or a computer with a network port for cable method.
+
+
+## Connecting to the CPE for the first time.
+
+### If the device has a management radio: Over WiFi
```

Comment:
Fix. Also fix all the Vale errors they are all valid including the passive, which can be written as active.